modules/mlflow/mlflow-fargate/stack.py

```diff
@@ -0,0 +1,206 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Any, List, Optional, cast
+
+from aws_cdk import Aspects, Duration, Stack, Tags
+from aws_cdk import aws_ec2 as ec2
+from aws_cdk import aws_ecr as ecr
+from aws_cdk import aws_ecs as ecs
+from aws_cdk import aws_ecs_patterns as ecs_patterns
+from aws_cdk import aws_efs as efs
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_s3 as s3
+from cdk_nag import AwsSolutionsChecks, NagPackSuppression, NagSuppressions
+from constructs import Construct, IConstruct
+
+
+class MlflowFargateStack(Stack):  # type: ignore
+    def __init__(
+        self,
+        scope: Construct,
+        id: str,
+        app_prefix: str,
+        vpc_id: str,
+        subnet_ids: List[str],
+        ecs_cluster_name: Optional[str],
+        service_name: Optional[str],
+        ecr_repo_name: str,
+        task_cpu_units: int,
+        task_memory_limit_mb: int,
+        autoscale_max_capacity: int,
+        artifacts_bucket_name: str,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        Tags.of(scope=cast(IConstruct, self)).add(key="Deployment", value=app_prefix[:64])
+
+        role = iam.Role(
```

Comment:
Do you want to name this role and prefix with the dep spec name to allow multiple instances of deployment of same module?

Reply:
I am relying on the autogenerated name that will be unique across stacks/deployments. Does that work?

Reply:
Yes, just did.